src/wm_info_provider/hyprland.rs

```diff
@@ -81,6 +81,15 @@ fn hyprland_cb(conn: &mut Connection<State>, state: &mut State) -> io::Result<()
         match hyprland.ipc.next_event() {
             Ok(event) => {
                 if let Some(active_ws) = event.strip_prefix("workspace>>") {
+                    hyprland.active_id = active_ws
+                        .parse()
+                        .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))?;
+                    updated = true;
+                } else if let Some(data) = event.strip_prefix("focusedmon>>") {
+                    let active_ws = &data[(data.find(',').ok_or_else(|| {
+                        io::Error::new(io::ErrorKind::InvalidData, "Too few fields in data")
+                    })? + 1)..];
```

Comment:
I think this can be better expressed using `data.split_once(',')`

Reply:
ah yes, that is nicer